UntitledPlatformerGameServer/GameServer.cs

```diff
@@ -242,8 +243,16 @@ void LoadGameObjects()
             loader.SpawnEnemy(_slowFactory, _globalEnemyId++, enemies, registry, collisionSystem);
             loader.SpawnEnemy(_fastFactory, _globalEnemyId++, enemies, registry, collisionSystem);
 
+            
+            GOCommand com = new BouncyPotionCommand(new TimeSpan(0, 0, 0, 20));
+
+            //Factory
+            Coin commonCoin = new CommonCoin(new GameObject(), com);

```

Comment:
Tavo patternas turetu atrodyt taip. Tik turek omeny kad teoriskai butu teisinga, bet istikro useless
```suggestion
           CoinFactory commonCoinFactory = new CommonCoinFactory();
           CoinFactory bigCoinFactory = new bigCoinFactory();
           
           Coin commonCoin = commoncoinFactory.CreateCoin();
           Coin bigCoin = bigCoinFactory.CreateCoin();
``

```